phoenix-core/src/main/java/org/apache/phoenix/parse/SelectStatement.java

```diff
@@ -267,6 +267,12 @@ public LimitNode getLimit() {
     }
     
     @Override
+    public Double getTableSamplingRate(){
+    	if(fromTable==null || !(fromTable instanceof ConcreteTableNode)) return null;
+    	return ((ConcreteTableNode)fromTable).getTableSamplingRate();
```

Comment:
Do we need this method? What happens in the case of a join, where there are multiple concrete tables?